Sources/SwiftDocC/Model/Rendering/RenderNode/RenderMetadata.swift

```diff
@@ -93,7 +93,7 @@ public struct RenderMetadata: VariantContainer {
     
     /// The variants of the title.
     public var titleVariants: VariantCollection<String?> = .init(defaultValue: nil)
-    
```

Comment:
Nit: Accidental white space change here it looks like. It's easier for us to track the impact of a PR in the git history over time if we only have changes that actually relate to the PR.